src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -1176,6 +1176,8 @@
   "MinimumFreeSpace": "Minimum Free Space",
   "MinimumFreeSpaceHelpText": "Prevent import if it would leave less than this amount of disk space available",
   "MinimumLimits": "Minimum Limits",
+  "MinimumUpgradeCustomFormatScore": "Minimum Upgrade Custom Format Score",
```

Comment:
I'm worried this name is too close to `Minimum Custom Format Score`, but more so that this particular verbiage is a little awkward, moving the `Upgrade` close to the end would help.

some ideas, but also curious what @stevietv might have in mind for ideas.

- `Custom Format Score Upgrade Differential`
- `Custom Format Score Upgrade Difference`
- `Minimum Custom Format Score Upgrade Differential`
- `Minimum Custom Format Score Upgrade Difference`

Reply:
I think right now its too similar in naming too. Id think something like `Minimum Custom Format Score Increase` - its like differential, just more simple.

Reply:
What about "Minimum Custom Format Improvement"?

Reply:
I'm in favor of using `Minimum Custom Format Improvement`. Would that be acceptable?

Reply:
Noticed that renaming to `Minimum Custom Format Score Improvement` will result in the name being displayed in three seperate rows, which looks too busy imo. Changed `Improvement` to `Increase` for now.

![image](https://github.com/user-attachments/assets/e13262aa-a853-4d3f-9fc8-805818887803)

Reply:
> Minimum Custom Format Score Upgrade Increment

This seems the clearest

Reply:
I think `Minimum Custom Format Score Upgrade Difference` fits better 🤔